packages/wrangler/src/index.tsx

```diff
@@ -211,10 +211,17 @@ export async function main(argv: string[]): Promise<void> {
     "init [name]",
     "📥 Create a wrangler.toml configuration file",
     (yargs) => {
-      return yargs.positional("name", {
-        describe: "The name of your worker.",
-        type: "string",
-      });
+      return yargs
+        .positional("name", {
+          describe: "The name of your worker.",
+          type: "string",
+        })
+        .option("yes", {
+          describe:
+            "Skip confirmation prompt, automatic yes to prompts; assume 'yes' as answer to all prompts and run non-interactively.",
```

Comment:
I think this needs bikeshedding but is non-blocking

Reply:
It's just way too long, let's keep it short and bikeshed. Maybe 'Answer "yes" to any prompts.'?